src/signerkey.js

```diff
@@ -89,4 +89,23 @@ export class SignerKey {
 
     return encodeCheck(strkeyType, raw);
   }
+
+  /**
+   * Creates a signed payload address (P...) from a signer (G...) and a payload.
+   *
+   * @param  {string} address  the signer of the signed payload, as a StrKey (G...)
+   * @param  {Buffer} payload  the raw payload that was signed
+   *
+   * @returns {string}   the StrKey-encoded signed payload address (P...)
+   */
+  static encodeSignedPayloadFromAddress(address, payload) {
```

Comment:
Any reason this can't have a shorter name?
```suggestion
  static encodeSignedPayload(address, payload) {
```

Reply:
probably not :shrug: that clashes with `StrKey.encodeSignedPayload`, but separate namespaces should be enough of a hint that they're different

Reply:
ooh how about `composeSignedPayload` :notes:

Reply:
What is `StrKey.encodeSignedPayload` for? It seems like a very low-level thing to expose to developers. We don't expose it in other SDKs.

Reply:
All of the `StrKey` functions in the SDK convert between raw buffers and strkeys. I'm not a fan and it deserves a refactor to line up more w/ the other SDK implementations, but that's historical.

Reply:
I think the historic reason was fine because once-upon-a-time all strkeys were just a single 32byte buffer.

It's unintuitive to me that a user must use a SignerKey object to generate a strkey. Maybe we should do soemthing like what we did with `MuxedAccount`s, where you create a `SignedPayloadSigner` which is an object that sits apart, and then you can generate a `SignerKey` from it.